kafka_consumer/main.go

```diff
@@ -71,6 +73,7 @@ func init() {
 	flag.StringVar(&ca, "ca", "", "CA certificate path for Kafka SSL connection")
 	flag.StringVar(&cert, "cert", "", "Certificate path for Kafka SSL connection")
 	flag.StringVar(&key, "key", "", "Private key path for Kafka SSL connection")
+	flag.IntVar(&maxBatchSize, "max-batch-size", 4096, "The limit of acceptable batch size in Open Protocol")
```

Comment:
should this flag set in sink-uri?

Reply:
We can put this into the sink-uri.

Reply:
I think this param should not exist in the cmd flag